File: checks/src/main/java/com/example/lint/checks/CustomIssueRegistry.java
package com.example.lint.checks;

import com.android.tools.lint.client.api.IssueRegistry;
import com.android.tools.lint.detector.api.Issue;

import org.jetbrains.annotations.NotNull;

import java.util.Arrays;
import java.util.List;

public class CustomIssueRegistry extends IssueRegistry {
    @NotNull
    @Override
    public List<Issue> getIssues() {
        return Arrays.asList(
                TimerTaskLeakDetector.ISSUE
        );
    }
}

File: checks/src/main/java/com/example/lint/checks/CustomIssueRegistry.kt
package com.example.lint.checks

import com.android.tools.lint.client.api.IssueRegistry
import com.android.tools.lint.detector.api.Issue
import java.util.*

class CustomIssueRegistry : IssueRegistry() {
    override val issues: List<Issue>
        get() = listOf(
            TimerTaskLeakDetector.ISSUE
        )
}